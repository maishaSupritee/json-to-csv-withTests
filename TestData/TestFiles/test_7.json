{
  "fruit": [
    {
      "P": "test\ncode",
      "I": null,
      "E": 10
    },
    {
      "P": "",
      "I": -10,
      "I": true
    }
  ]
}
