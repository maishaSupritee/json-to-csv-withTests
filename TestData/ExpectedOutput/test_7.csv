"fruit_P","fruit_I","fruit_E"

"test
code", "None", "10"

"","True",""

